release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -647,6 +654,18 @@ As an administrator, you can now install the Network Observability Operator to o
 
 For more information, see xref:../networking/network_observability/network-observability-overview.html#network-observability-overview[Network Observability].
 
+[id="ocp-4-12-nw-shiftstack-ipv6-secondary"]
+==== IPv6 for secondary network interfaces on {rh-openstack}
+
+IPv6 for secondary network intefaces is now supported in clusters that run on {rh-openstack}. 
+
+// TODO: For more information,see ${DOC_WHEN_MERGED} 
+
+[id="ocp-4-12-nw-shiftstack-udp-load-balancing"]
+==== UDP support for load balancers on {rh-openstack}
+
+UDP is now supported for load balancers for clusters that run on {rh-openstack}.
```

Comment:
I'd rephrase it into "UDP protocol". But anyway in general we should say that OpenStack is now using the external cloud provider (cloud-controller-manager/CCM). This will tell way more for people actually knowing about Kubernetes. UDP LBs support is just one of the features we get by using CCM. Another one is e.g. ability to use OVN Octavia provider.

Reply:
```suggestion
Resulting from the switch to an external cloud provider, UDP is now supported for load balancers for clusters that run on {rh-openstack}.
```

@dulek Am I wrong, or is the "P" in "UDP" for "Protocol?"